README.md

```diff
@@ -82,6 +82,14 @@ conf file as desired. Files can be local as above, or available via http/https.
 
 Also see the additional usage section at the bottom for additional details.
 
+Typecasting values
+------------------
+
+It's possible to add field type info to the field name. By default, all fields will have `type=String`.
```

Comment:
Nitpick but how about this:
By default, using op of mean`,`max`,`sum`, `div2` or `min` will result in `type=Float`, using `self` will get the type from the annotation VCF and other fields will have `type=String`

and drop the final sentence from this changeset.